### PR TITLE
DDF-1279: reorganized documentation

### PR DESCRIPTION
**Pull request by ricklarsen**
@clockard @vinamartin @stustison @gordocanchola @oscaritoro

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/codice/ddf/pull/367" target="_blank">ReviewNinja

File: distribution/docs/src/main/resources/IntContents.adoc [hunk 1]
```diff
@@ -1,10 +1,245 @@
...
+.. look up the GeoJson Input Transformer based on the mime type of the ingested file,
+.. create a metacard based on the metadata parsed from the ingested GeoJson file, and
+.. insert the metacard into the Catalog using the CatalogFramework.
++
+Note that XML metadata for text searching is not automatically generated from GeoJson fields.
+. Verify GeoJson file was stored using the Content REST endpoint.
+.. Install the feature content-rest-endpoint using the Features tab in the Web Console.
+.. Send a GET command to read the content from the content repository using the Content REST endpoint. This can be done using curl command below. Note that the GUID will be different for each ingest. The GUID can be determined by           going to the <DISTRIBUTION_INSTALL_DIR>/content/store directory and copying the sub-directory in this folder (there should only be one).
+
+.*NIX Example
```
Comment (vinamartin) [inline distribution/docs/src/main/resources/IntContents.adoc]: There's a *NIX example given here but no Windows one although a similar action is performed above. Should one be added here?

File: distribution/docs/src/main/resources/IntContents.adoc [hunk 2]
```diff
@@ -1,10 +1,245 @@
...
+Other applications may be installed at a later time.
+====
++
+. After the installation has been configured the instance should be restarted.
+. Go to https://localhost:8993/services and verify five REST services are available: admin, application, metrics, catalog, and catalog/query.
+. Click on the links to each REST service's WADL to see its interface.
+. In the Admin Console (at /admin), configure the system settings.
+.. Enter the username of "admin" (no quotes) and the password "admin" (no quotes).
+.. Select Platform app
+.. Select Platform Global Configuration.
```
Comment (clockard) [inline distribution/docs/src/main/resources/IntContents.adoc]: The Platform Global Configuration no longer exists. This is configured in the installer now or through the properties file. I believe there is a ticket out to  make this configurable via the admin ui including the restart.

File: distribution/docs/src/main/resources/DDFOmniContents.adoc [hunk 3]
```diff
@@ -7,66 +75,92 @@ include::ManContents.adoc[]
...
+== Managing {branding} UI
+
 include::${project.build.directory}/apps/ui/ManContents.adoc[]
 
 <<<
 
 == Integrating {branding}
 
-<<<
-
```
Comment (oscaritoro) [inline distribution/docs/src/main/resources/DDFOmniContents.adoc]: Still learning about adocs... is this one ok without a path (like ${project.build.directory}/IntContents.adoc[]) or "title" in the == Integrating  {branding} above?

File: distribution/docs/src/main/resources/IntContents.adoc (hunk 1 above)
Comment (oscaritoro) [inline distribution/docs/src/main/resources/IntContents.adoc]: Agreed, even though the command might be the same (not sure you even need to specify the .exe) in this case, a precedent was kind of set to show Windows Example... Also, has anyone "tested" these commands recently on *nix and/or Windows?

Comment (ricklarsen) [inline distribution/docs/src/main/resources/IntContents.adoc]: Per my research, the '.exe' is not needed, which makes these commands both identical, as no file path is being specified. The port number is a holdover from a past edit and is now fixed.

File: distribution/docs/src/main/resources/IntContents.adoc (hunk 2 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/IntContents.adoc]: Fixed.

File: distribution/docs/src/main/resources/DDFOmniContents.adoc (hunk 3 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/DDFOmniContents.adoc]: Yes the ${project.build.directory} gets resolved by Maven filtering before the docs are rendered.
The {branding} gets rendered by asciidoctor, even in titles.
(the $ is significant for this reason)

**Comment by vinamartin** [comment]
![Thumbs up]
(http://www.quickmeme.com/img/10/10e7950c3f1af890b24161e0aa5babfa53744a91c4cbc6473f50d296fea6282d.jpg)

**Pull request merged**